RELEASES.md

```diff
@@ -1,3 +1,120 @@
+Version 1.57.0 (2021-12-02)
+==========================
+
+Language
+--------
+
+- [Support `#[track_caller]` on closures and generators][87064]
```

Comment:
I believe this is actually still feature-gated currently.

Reply:
Ah, ok, removed.